shap/__init__.py

```diff
@@ -1,6 +1,6 @@
 # flake8: noqa
 
-__version__ = '0.28.5'
+__version__ = '0.28.5s'
```

Comment:
I was trying to look up PEP standard on versioning https://www.python.org/dev/peps/pep-0440/

It seems the closest we can do if we want to be compliant, is to use the Post Release format `postN`.

But either way, if we want to go with `s` (I assume for Shopify), I think we should explicitky spell out `shopify`.... so `-shopify, .shopify` maybe. What do you think?

Reply:
Ooh `post1` makes sense I think. I was looking up the standard semver docs (https://semver.org/) and they didn't seem to have any relevant conventions.